hack/update-codegen.sh

```diff
@@ -37,3 +37,12 @@ for group in work; do
     --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.txt \
     ${verify}
 done
+
+for group in nucleus; do
```

Comment:
make line 24 in this file say `for group in cluster work nucleus; do` and remove the duplicate blocks